lib/common/presentation/views/referencing/referencing.dart

```diff
@@ -0,0 +1,316 @@
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+/// The page [Referencing]
+class Referencing extends StatelessWidget {
+  /// Constructs an instance of [FaqScreen]
+  const Referencing({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) => Container(
+        padding: const EdgeInsets.all(40.0),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            Row(
+              children: [
+                Image.asset(
+                  'assets/images/referencingimage.png',
+                  height: 100,
+                  width: 100,
+                ),
+                const SizedBox(width: 20),
+                const Text(
+                  'Referencing & Citation',
+                  style: TextStyle(fontSize: 40, fontWeight: FontWeight.bold),
+                ),
+              ],
+            ),
+            const SizedBox(height: 40),
+            const Text(
+              'When writing assignments, reports, research papers or theses, '
+              'you must acknowledge quotations, information and ideas taken '
+              'from other authors through a bibliography or list of '
+              'references.',
+              style: TextStyle(fontSize: 15),
+            ),
+            const SizedBox(height: 20),
+            Row(
+              children: [
+                const Text(
+                    'The Library has developed guides for the citation styles '
+                    'most commonly used within the University. Use of these '
+                    'guides is supported in the ',
+                    style: TextStyle(fontSize: 15)),
+                InkWell(
+                  onTap: () => launch(
+                      'https://policies.westernsydney.edu.au/download.php?associated=1&id=716&version=1'),
+                  child: Text('University Referencing Styles Guidelines.',
+                      style: TextStyle(
+                        fontSize: 15,
+                        color: Colors.red[900],
+                        decoration: TextDecoration.underline,
+                      )),
```

Comment:
When it comes to colours, to stay consistent use the app's defined theme. 

This applies to anything style related, textStyles, constraints colours etc